.circleci/config.yml

```diff
@@ -65,7 +65,7 @@ jobs:
       - run:
           name: Check file paths
           command: |
-            if git diff --name-only main HEAD | grep -E 'app/experimenter/targeting|app/experimenter/experiments'
+            if git diff --name-only main HEAD | grep -E 'app/experimenter/targeting|app/experimenter/experiments|app/tests'
```

Comment:
I don't think we should change this. We shouldn't need to run the integration tests against beta/release for every commit.

Reply:
What do you mean on every commit?

Reply:
@yashika-khurana Why did you take this out?

Reply:
oh I read the above @jrbenny35 as he suggested, I can bring it back

Reply:
Right but adding `app/tests` won't cause it to run on every commit, only changes to the tests folder, which is specifically what this ticket is requesting.  Just because someone makes a suggestion in a PR does not mean it should be followed, and @jrbenny35 I would recommend framing feedback on PRs as questions rather than statements in general.

Reply:
@jaredlockhart its back now

Reply:
> Right but adding `app/tests` won't cause it to run on every commit, only changes to the tests folder, which is specifically what this ticket is requesting. Just because someone makes a suggestion in a PR does not mean it should be followed, and @jrbenny35 I would recommend framing feedback on PRs as questions rather than statements in general.

Sorry, was sort of in a rush, should've saved my feedback for a bit later. So yes, it wont run on every commit but running the tests for beta/release for changes in app/tests is a not needed in my opinion. Running the tests against nightly should be fine but we also don't do that many changes to app/tests. 

Another point here is the path should be ```app/tests/integration/nimbus``` as we have a separate job for ```app/tests/integration/legacy```

Reply:
@jrbenny35 specifically the issue here is that changes in `app/tests` including:
 
- dependency changes
- test file changes

can break the tests, which is what just happened and resulted in the integration tests being broken in main.  So we could say that the beta/release tests are so close to the nightly tests that if the nightly tests pass we expect the beta/release tests to work, but there could still be cases where one passes and the other fails, so the goal here is to say ANY changes to the tests folder should force ALL integration tests to run so we don't introduce another breaking regression into main.